extras/indexing/src/main/java/mvm/rya/indexing/accumulo/ConfigUtils.java

```diff
@@ -118,22 +118,22 @@
     public static final String GEO_PREDICATES_LIST = "sc.geo.predicates";
 
     public static final String TEMPORAL_PREDICATES_LIST = "sc.temporal.predicates";
-    
+
     public static final String USE_MONGO = "sc.useMongo";
 
-    public static boolean isDisplayQueryPlan(Configuration conf){
+    public static boolean isDisplayQueryPlan(final Configuration conf){
         return conf.getBoolean(DISPLAY_QUERY_PLAN, false);
     }
-    
+
     /**
      * get a value from the configuration file and throw an exception if the value does not exist.
-     * 
+     *
      * @param conf
      * @param key
      * @return
      */
-    private static String getStringCheckSet(Configuration conf, String key) {
-        String value = conf.get(key);
+    private static String getStringCheckSet(final Configuration conf, final String key) {
```

Comment:
don't add final for the sake of adding final.  makes it hard to see what really changed.